tests/commands/test_build.py

```diff
@@ -596,6 +595,29 @@ def test_build_generics_tree(tmp_path, caplog):
     assert not (build_dir / 'dir2' / 'dir4').exists()
     assert (build_dir / 'dir2' / 'dir5').exists()
 
+    if expect_hardlinks:
+        # they're hard links
+        assert (build_dir / 'crazycharm.py').samefile(entrypoint)
+        assert (build_dir / 'file1.txt').samefile(file1)
+        assert (build_dir / 'dir2' / 'file2.txt').samefile(file2)
+    else:
+        # they're *not* hard links
+        assert not (build_dir / 'crazycharm.py').samefile(entrypoint)
+        assert not (build_dir / 'file1.txt').samefile(file1)
+        assert not (build_dir / 'dir2' / 'file2.txt').samefile(file2)
```

Comment:
Please check that the files has the same content, permissions, etc... thanks!

Reply:
picky, picky...! :-D